src/polar/Pie.tsx

```diff
@@ -479,6 +479,7 @@ export class Pie extends PureComponent<Props, State> {
   renderSectorsStatically(sectors: PieSectorDataItem[]) {
     const { activeShape, blendStroke, inactiveShape: inactiveShapeProp } = this.props;
     return sectors.map((entry, i) => {
+      if (entry?.startAngle === 0 && entry?.endAngle === 0 && sectors.length !== 1) return null;
```

Comment:
why `sectors.length !== 1`? Is 1 when animation is starting?

Are the angle values always going to be 0 or could they be undefined? If they're undefined this will go through

Reply:
Hello 👋 
> why sectors.length !== 1? Is 1 when animation is starting?   

when there is 1 sector, there is 1 pie "slice". With 1 pie sector/slice, there will not be a duplicate key error, as there will not be multiple sectors/slices.  
The animation starting is represented by the sector(s) startAngle & endAngle both being `0`.  

> Are the angle values always going to be 0 or could they be undefined? If they're undefined this will go through  

The angle values are never `undefined` as far as I can tell 🤔 The angles change, during an animation, from 0 to the angle of the pie slice, "tick-by-tick".   

The duplicate-key error appears in lets say an animated pie with 3 slices:
- at the beginning of the animation, all 3 slices have the same startAngle of 0
- at the beginning of the animation, all 3 slices have the same endAngle of 0
- the angle(s) of each separate slice are the (only?!) uniquely-identifying attributes that can be used to associate each item with a unique key
- Due to the animation of each slice starting with each slice angle value at `0`, the beginning of the animation throws the duplicate key error. Once the animation gets past `0` values for each slice, the duplicate key error goes away during the rest of the animation.

Reply:
Thanks for the detailed explanation 🚀